salt/utils/gitfs.py

```diff
@@ -1549,10 +1549,10 @@ def _perform_checkout(checkout_ref, branch=True):
 
             elif tag_ref in refs:
                 tag_obj = self.repo.revparse_single(tag_ref)
-                if not isinstance(tag_obj, pygit2.Commit):
+                if True not in [isinstance(tag_obj, pygit_type) for pygit_type in [pygit2.Commit, pygit2.Tag]]:
```

Comment:
`isinstance` supports tuples. Could you please update this line accordingly?
```python
if not isinstance(tag_obj, (pygit2.Commit, pygit2.Tag)):
```

Reply:
Done